plugins/modules/gitlab_instance_variable.py

```diff
@@ -64,6 +64,12 @@
           - The variable value.
           - Required when O(state=present).
         type: str
+      description:
+        description:
+          - A description for the variable.
+          - Support for descriptions requires GitLab >= 16.8.
```

Comment:
just double checking, the minimum version here is indeed different from the other ones?

Reply:
That's correct, instance variable descriptions came later.

Instance: https://gitlab.com/gitlab-org/gitlab/-/merge_requests/140549
Project, Group: https://gitlab.com/gitlab-org/gitlab/-/merge_requests/124285